src/server/ideController.py

```diff
@@ -428,12 +432,14 @@ def _save_callback(self, filename, changes, version, users):
                            else create_change_remove_element_dict(element))
                           for element in changes]
 
-    message_sent = simplejson.dumps(wrap_opCode('save',
-                                                create_file_version_dict(filename,
-                                                                         version,
-                                                                         serialized_changes)))
-
     for user in users:
+      for sc in serialized_changes:
+        sc['author'] = (sc['author'] == user)
```

Comment:
Encapsulate this function near the others (create_change_add_element_dict)
so that if the author key changes this function would not be too far